src/lib/Form/DataTransformer/TrashItemTransformer.php

```diff
@@ -66,7 +66,7 @@ public function reverseTransform($value): ?TrashItem
         }
 
```

Comment:
It would be nice to add here 
```
        if (!ctype_digit($value)) {
            throw new TransformationFailedException('Expected a numeric string.');
        }
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1127/commits/43fc63f6bbcfe91a8af66f3a4f73ee339db0d19d